src/chains/arbitrum/signatureTypes.ts

```diff
@@ -0,0 +1,110 @@
+import { SignatureType } from '@/chains';
+import { signatureTypes as mainnetSignatureTypes } from '@/chains/mainnet/signatureTypes';
+
+const txTypeDocs = 'https://developer.arbitrum.io/arbos/geth#transaction-types';
```

Comment:
Thought I noticed a gap, and there is indeed a tx type 103 (`0x067`) that was removed but existed previously. Not sure if it ended up in a mainnet deployment. It was the "ArbitrumWrappedTxType". Worth documenting maybe?

Reply:
Tracking this in https://github.com/mds1/evm-diff/issues/17